Xamarin.Forms.Platform.WPF/Renderers/NavigationPageRenderer.cs

```diff
@@ -16,7 +16,8 @@ public FormsLightNavigationPage(NavigationPage navigationPage)
 
 		public override void OnBackButtonPressed()
 		{
-			NavigationPage.PopAsync();
+			if (NavigationPage.CurrentPage?.SendBackButtonPressed() == false)
```

Comment:
!NavigationPage.CurrentPage?.SendBackButtonPressed() ?

Reply:
I think we cannot use (!) when there is a nullable check (CurrentPage?)

Reply:
You're right it's not very clean. It would be better to do this:

if(Navigation?.CurrentPage == null)
return ;

if (!NavigationPage.CurrentPage.SendBackButtonPressed())
NavigationPage.PopAsync();

I can modify it like that